var/spack/repos/builtin/packages/py-cryptography/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+from spack import *
+
+
+class PyCryptography(PythonPackage):
+    """cryptography is a package which provides cryptographic recipes
+       and primitives to Python developers"""
+
+    homepage = "https://pypi.python.org/pypi/cryptography"
+    url      = "https://pypi.python.org/packages/ec/5f/d5bc241d06665eed93cd8d3aa7198024ce7833af7a67f6dc92df94e00588/cryptography-1.8.1.tar.gz#md5=9f28a9c141995cd2300d0976b4fac3fb"
+
+    version('1.8.1', '9f28a9c141995cd2300d0976b4fac3fb')
+
+    depends_on('py-setuptools', type=('build', 'link'))
+    depends_on('py-cffi',       type=('build', 'link', 'run'))
+    depends_on('py-asn1crypto', type=('build', 'link', 'run'))
```

Comment:
These should be `'build', 'run'`. Also, you're missing half a dozen dependencies. Check the `setup.py` and look at `install_requires` and `setup_requires`.